testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -200,3 +202,19 @@ def forward(self, x):
                 )
 input = Variable(torch.randn(1, 3, 32, 32))
 save_data_and_model("constant", input, model)
+
+
+class Transpose(nn.Module):
+
+    def __init__(self):
+        super(Transpose, self).__init__()
+        self.squeeze1 = nn.Linear(3, 4, bias=True)
+
+    def forward(self, x):
+        x = self.squeeze1(x)
+        return torch.t(x)
+
+
+input = Variable(torch.randn(2, 3))
+model = Transpose()
+save_data_and_model("transpose", input, model)
```

Comment:
@l-bat, looks like we need to add this test into `test_onnx_importer.cpp`.

Reply:
done